### ISSUE TITLE
How to compile examples/simple_parse.c ?

### ISSUE THREAD

**Issue by JlnWntr** [akheron/jansson#355]
Compilation-instructions are needed for the simple_parse-C-program. gcc complains about too many missing function-definitions. Compiling simple_parse.c does not really seem simple.

**Comment by Mephistophiles** [comment]
This example is compiled automatically when compiling the Jansson. If you have not configured JANSSON_EXAMPLES to OFF when you are configuring jansson.

**Comment by akheron** [comment]
Yeah, it is when compiling with CMake. With ./configure && make, this doesn't seem to be the case currently.

**Comment by harkue** [comment]
You can also see this comment if you want to compile the example manually:
[#214](https://github.com/akheron/jansson/pull/214#issuecomment-68262493)
